hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+    - -19.6 dBm at 315 MHz
+    - -14.4 dBm at 433 MHz
+    - -13.3 dBm at 470 MHz
+
+  - Tx bandwidth requirement
+
+    The 20dB emission bandwidth should be no wider than 0.25% of the center operation frequency.
+
+  - Spurs emission limits
+    General requirement is very loose in this part, the limit is -20dBc everywhere. But for the spurs lie in restrict bands defined in Part-15.205, the spurs including harmonics will be less than -49dBm for frequency < 960MHz, and be less than -41dBm for frequency >960MHz.
+    Since the fundamental power is very low, the harmonics limits can be achieved very easy by only adding a 3-order low pass filter in the RF path.
+
+  - Rx requirements
+    
+    No sensitivity requirement. But the limits for harmonics or spurs in a restricted band is defined in “FCC Part 15.109 Unintentional Radiators” as following:
+    - Spurious falls below 960 MHz = -49.2 dBm EIRP (max) (15.209.a);
+    - Spurious falls above 960 MHz = -41.2 dBm EIRP (max) (15.209.a).
+
+- Part-15.247
+  Typical applications are Weather station reporting, Automatic meter reading, and long range communication in this part. It covers wide bandwidth applications in 902~928MHz frequency band:
+  a)Frequency Hopping Spread Spectrum (FHSS);
+  b)Wideband digital modulation (BWMOD_6dB > 500 kHz);
+  c)Direct Sequence Spread Spectrum; 
+
+  - Tx power limits
+    Since it is a wide band appliction in this regulation. The maximum output power can be high since the band "spread out" effection. The fundamental output power should be a Peak measurement.
+     - Wideband digital modulation → 1 watt (+30 dBm) max;
+     - FHSS (> 50 hopping channels) → 1 watt (+30 dBm) max;
+     - FHSS (> 25 hopping channels) → 0.25 watt (+24 dBm) max.
+
+  - Tx bandwidth requirement
+     For FHSS operation system, the maximum allowed 20dB bandwidth of hopping channel is 500KHz. But for using digital modulation techniques, the minimum 6dB bandwidth shall be at least 500KHz.
+   
+
+  - Power spectral density(PSD) limit
+    This requirement is for digitally modulated system. The PSD should not be greater than 8dBm in any 3KHz band during any time interval of continuous transmission.
+
+  - Spurs emission limits
+    General requirement is very loose in this part, the limit is -20dBc everywhere. But for the spurs lie in restrict bands defined in Part-15.205, the spurs including harmonics will be less than -49dBm for frequency < 960MHz, and be less than -41dBm for frequency >960MHz. For the harmonics limits, the strength limit can be relax by using DCF factor. Because radiated spurious emissions above 1 GHz are AVERAGE measurements covered by FCC Part 15.35.
+    Basically it needs higher order(5-order) low pass filter since the fundamental power is very high(+30dBm).
```

Comment:
Add spaces to change "order(5-order)" to "order (5-order)" and "high(+30dBm)" to "high (+30dBm)"

Reply:
Corrected.